package.json

```diff
@@ -42,6 +42,7 @@
     "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@fortawesome/react-fontawesome": "0.2.0",
     "core-js": "3.32.1",
+    "moment": "^2.29.2",
```

Comment:
Why we need it, why don't use @edx/frontend-platform/i18?

Reply:
@edx/frontend-platform/i18n allows to format dates but as a string internationalizing them, but I needed to compare the dates. With this in mind, I chose "moment" over the native way because the first gives a little bit more readability (it is not mandatory, and the change is pretty small).